entities/bizaiChats.json

```diff
@@ -0,0 +1,56 @@
+{
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "uid of the chat",
+      "example": "d290f1ee-6c54-4b01-90e6-d701748f0851"
+    },
+    "actor_uid": {
+      "type": "string",
+      "description": "uid of the actor that created the chat",
+      "example": "ghp0f1ee-6c54-4b01-90e6-d701748f0851"
+    },
+    "agent": {
+      "type": "string",
+      "description": "Agent associated with the chat",
+      "example": "vanilla"
+    },
+    "metadata": {
```

Comment:
Is this the system message? if it is than metadata is not a good name. lets discuss this